src/OrchardCore.Modules/OrchardCore.Lists/Feeds/ListFeedQuery.cs

```diff
@@ -79,8 +79,7 @@ public async Task ExecuteAsync(FeedContext context)
             else
             {
                 context.Builder.AddProperty(context, null, "title", contentItem.DisplayText);
-
-                context.Builder.AddProperty(context, null, "description", bodyAspect.Body != null ? $"<![CDATA[{bodyAspect.Body?.ToString()}]]>" : String.Empty);
+                context.Builder.AddProperty(context, null, "description", new XCData(bodyAspect.Body?.ToString() ?? String.Empty).ToString());
```

Comment:
Same issue in the "else"  as for the CommonFeedItemBuilder here too.

Reply:
Also no need to out in empty CDATA if the body is empty.

Reply:
BTW I think there's no issue with `<![CDATA[]]>` I see this quite often

Reply:
> BTW I think there's no issue with `<![CDATA[]]>` I see this quite often

Correct there is no issue... personal preference ;)

Reply:
It is better to have `<![CDATA[]]>` than not, as it enforce defining which type of data is contained in that parameter even if it's empty. If you use a "" value in that attribute then it means that the returned content could vary of type.